app/views/products/index.html.haml

```diff
@@ -8,7 +8,7 @@
       .panel
         .panel-body
           .alert.alert-danger{ ng: { if: "productErrors" }}
-            %ul{ ng: { repeat: "val in productErrors" }}
+            %ul{ ng: { repeat: "product in products" }}
```

Comment:
repeat 'product in $scope.products' if there are $scope.productErrors, then show $scope.val?